Formula/bash-completion.rb

```diff
@@ -1,5 +1,5 @@
 # NOTE: version 2.0 is out, but it requires Bash 4, and macOS ships
-# with 3.2.48. See homebrew-versions for a 2.0 formula.
+# with 3.2.48. If you've upgraded bash, use bash-completion@2 instead.
```

Comment:
Indeed! I’ve also updated the line above to just say “version 2”, since 2.0 feels outdated, and there’s little point in keeping it fully up-to-date whenever newer versions are out.